settings.py

```diff
@@ -812,6 +813,13 @@
 
 BITLY_OAUTH_TOKEN = None
 
+OAUTH2_PROVIDER = {
+    'SCOPES': {
+        'access_apis': 'Access API data on all your CommCare projects',
```

Comment:
Since at this point there's only one scope and only one reference to that scope, it's probably fine to leave like this, but if/when it expands, you might want to refer to scopes by reference (ie a series of constants or an enum type)